opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -743,7 +743,7 @@ type Work {
   received_time: DateTime
   processed_time: DateTime
   completed_time: DateTime
-  completed_number: Int
+  completed_number: String
```

Comment:
are you sure about this?
Might collide with Julien's work on #5762  ?

Reply:
Let me try it out.

Reply:
Since the previous error is no longer logged, I have rollbacked this modification